test/table.template

```diff
@@ -0,0 +1,76 @@
+  CREATE TABLE "?"."LOCALUSER1"
+   (    "ID" VARCHAR(20) NOT NULL,
```

Comment:
OK.  I dont' think we need it. but it doesn't hurt to have it since it's coming from the original file.